mern/client/src/screens/RecoverSession/recoverSession.js

```diff
@@ -6,23 +6,24 @@ import { useNavigate } from "react-router-dom";
 
 export default function RecoverSession() {
   const Navigate = useNavigate();
-  const [sessionid, setSessionID] = useState("");
-  const [, setNavState] = useContext(SessionContext);
+  const [, setSessionID, , ] = useContext(SessionContext);
+  const [inputText, setInputText] = useState("")
 
   const PageChange = (url) => {
     Navigate(url);
   };
 
   const handleSubmit = (event) => {
+    // prevent page reload
     event.preventDefault();
     // show navbar buttons for starting-intent page
-    setNavState(true);
-    alert(`The session ID you entered was: ${sessionid}`);
-  };
+    setSessionID(inputText);
+    PageChange("/startingintent")
```

Comment:
I am just wondering, would this take the users to the first intents based on the transcript or the point they saved last time?